pkg/storage/queue.go

```diff
@@ -531,6 +531,10 @@ func (bq *baseQueue) processReplica(
 				log.VEventf(queueCtx, 3, "not holding lease; skipping")
 				return nil
 			}
+			if _, harmless := err.GetDetail().(*roachpb.RangeNotFoundError); harmless {
```

Comment:
ah, vivek is right, we should probably make this a type-switch and combine with the above.